upload_events.py

```diff
@@ -25,9 +31,10 @@
 @click.option("--project-id", type=int, help="the project id to be dumped, alternative to dsn, or (project-slug,org-slug)")
 @click.option("--project-slug", type=str, help="project slug, must also use org-slug, alternatively use project-id or dsn")
 @click.option("--org-slug", type=str, help="organization slug, must also use project-slug, alternatively use project-id or dsn")
-@click.option("--expire-days", default=0, type=int, help="sets timestamp in the past to influence retention")
+@click.option("--network-threads", default=64, help="How many threads to use for sending")
```

Comment:
In my tests, 2 * num_cpu seemed to get the fastest processing time.

Reply:
yeah I still have to tweak this number in prod. 64 doesn't harm really, and we need to throttle upload anyway.